### PR TITLE
Make the CMS teams less specific

### PR DESCRIPTION
**Pull request by rustydb**
### Summary and Scope

- Fixes: Waiting on CODEOWNERS for CMS even though CMS has already approved a PR.

#### Issue Type

- Docs Pull Request

CMS is broken up into 13 teams in GitHub. More often than not a qualified CMS team member will approve a PR but they won't exist in the sub-team specified in CODEOWNERS.

This change replaces the code owner for CMS items with the parent team, cray-management-systems.

### Prerequisites

- [ ] I have included documentation in my PR (or it is not required)
- [ ] I tested this on internal system (if yes, please include results or a description of the test)
- [ ] I tested this on a vshasta system (if yes, please include results or a description of the test)

### CODE REVIEW

**Review by dlaine-hpe — APPROVED** [review]

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 /cray/modules/artifacts @Cray-HPE/cray-management-systems
 /cray/modules/badger    @Cray-HPE/hardware-management
-/cray/modules/bos       @Cray-HPE/cms-core-bos
+/cray/modules/bos       @Cray-HPE/cray-management-systems
 /cray/modules/bss       @Cray-HPE/hardware-management
 /cray/modules/capmc     @Cray-HPE/hardware-management
-/cray/modules/cfs       @Cray-HPE/cms-core-cfs
+/cray/modules/cfs       @Cray-HPE/cray-management-systems
 /cray/modules/config    @Cray-HPE/cloud
 /cray/modules/cps       @Cray-HPE/cray-management-systems
```
Comment (mharding-hpe) [inline .github/CODEOWNERS:9]:
```suggestion
```

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,15 +1,15 @@
 /cray/modules/ars       @Cray-HPE/cray-management-systems
 /cray/modules/artifacts @Cray-HPE/cray-management-systems
```
Comment (mharding-hpe) [inline .github/CODEOWNERS:2]: I think PET owns Ceph/S3, right?

```suggestion
/cray/modules/artifacts @Cray-HPE/platform-engineering
```

Comment (mharding-hpe) [inline .github/CODEOWNERS:2]: I think PET owns Ceph/S3, right?

```suggestion
/cray/modules/artifacts @Cray-HPE/platform-engineering
```

File: .github/CODEOWNERS (hunk 1 above)
Comment (mharding-hpe) [inline .github/CODEOWNERS:9]: This isn't owned by CMS. I don't know that any of our teams really owns it, so it should probably be moved to that bottom section with the other product-specific stuff

**Review by mharding-hpe — CHANGES_REQUESTED** [review]

File: .github/CODEOWNERS (hunk 2 above)
Comment (rustydb) [reply .github/CODEOWNERS:2]: Makes sense to me.

**Pull request merged**